.clang-tidy

```diff
@@ -27,6 +27,8 @@ Checks: [
   '-bugprone-not-null-terminated-result',
   '-bugprone-reserved-identifier', # useful but too slow, TODO retry when https://reviews.llvm.org/rG1c282052624f9d0bd273bde0b47b30c96699c6c7 is merged
   '-bugprone-unchecked-optional-access',
+  '-bugprone-crtp-constructor-accessibility',
+  '-bugprone-suspicious-stringview-data-usage',
```

Comment:
Could you please explain why these are ignored? They looks good, especially the second one.

Reply:
I was fixing another problem with clang-tidy, and part of the work was compiling it on my machine. But I have the latest clang-19 instead of clang-18, and some other new checks appeared. They were non-trivial to fix, and also I remember how I tried to do it earlier but didn't finish. That's why we disable these checks first.